### PR TITLE
fix get intent bad input (1,4)

### PR DESCRIPTION
**Pull request by haixoay96**
**Proposed changes**:
- ...fix get intents label

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
thanks for submitting this PR, we'll give it a review as soon as possible

**Comment by Ghostvv** [comment]
Do you mind signing CLA?

**Comment by Ghostvv** [comment]
Could you please describe the error that you hav e with the old code? Because the next line gets rid of empty dimensions: https://github.com/RasaHQ/rasa_nlu/blob/a0421db8a2566f3e3a22897042a96be5e679714d/rasa_nlu/classifiers/sklearn_intent_classifier.py#L184

**Comment by haixoay96** [comment]
> yes, and then `intents.flatten()` transforms it to `[1 2 3 4]`

>But  in line 180, 181
   intent_ids, probabilities = self.predict(X)
   intents = self.transform_labels_num2str(intent_ids)
   intent_ids will is [[1 2 3 4]] should throw exception bad input (1,4) when you call 181. intent_ids should is [1 2 3 4]

**Comment by haixoay96** [comment]
refe to https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html
Examples

LabelEncoder can be used to normalize labels.

>>>
>>> from sklearn import preprocessing
>>> le = preprocessing.LabelEncoder()
>>> le.fit([1, 2, 2, 6])
LabelEncoder()
>>> le.classes_
array([1, 2, 6])
>>> le.transform([1, 1, 2, 6]) 
array([0, 0, 1, 2]...)
>>> le.inverse_transform([0, 0, 1, 2])
array([1, 1, 2, 6])
